src/sidebar/components/test/menu-item-test.js

```diff
@@ -113,9 +119,12 @@ describe('MenuItem', () => {
   });
 
   describe('submenu', () => {
+    afterEach(() => {});
+
     it('shows the submenu indicator if `isSubmenuVisible` is a boolean', () => {
       const wrapper = createMenuItem({
         isSubmenuVisible: true,
+        submenu: <div role="menuitem">Submenu content</div>,
```

Comment:
Some of these tests were causing prop error warnings because `children` is required for the `menu-keyboard-navigation` component